### ISSUE TITLE
A bigger example, following more of the guide structure?

### ISSUE THREAD

**Issue by zeroasterisk** [studiointeract/tracker-component#10]
I really like what you've built here.

But I also really like the new guide structure of splitting out all of my meteor code to `containers`.

Can you provide a bigger example, with things split out as in the guide?

_(thank you!!!)_

**Comment by timbrandin** [comment]
Yeah, sure @zeroasterisk, I'm working on a Slack Chat app example, coming soon :)

**Comment by zeroasterisk** [comment]
Sweet - thanks!  Also, FYI, I'm also a fan of the stuff @ https://github.com/themeteorchef/base - might be worth discussing switching them to your container layer too...?
